Product/Production/Adapters/General/CONNECTAdminGUI/src/main/webapp/certificateManager.xhtml

```diff
@@ -657,6 +657,8 @@
                                         </p:messages>
                                         <div class="content">
                                             <h:outputText>To override the existing CA Root and CA Intermediate in the Truststore: select certificate and upload the CA chain.</h:outputText>
+                                            <br />
+                                            <h:outputText>Upon completing the import wizard the new KeyStore and TrustStore will be created under `importWizard/new` and the old KeyStore and TrustStore will be backup under `importWizard/backup`. The user must manually replace the existing KeyStore and TrustStore, and restart server to complete the process.</h:outputText>
```

Comment:
Why do we have a total of 3 folders for this? I thought the criteria was to have everything under one folder.